modules/thermal_hydraulics/src/bcs/HeatStructure2DRadiationCouplerRZBC.C

```diff
@@ -53,6 +55,7 @@ ADReal
 HeatStructure2DRadiationCouplerRZBC::computeQpResidual()
 {
   const auto T_coupled = computeCoupledTemperature();
+  const Real circumference = computeCircumference(_q_point[_qp]);
   return _sigma * (std::pow(_u[_qp], 4) - std::pow(T_coupled, 4)) / _radiation_resistance *
-         _perimeter * _test[_i][_qp];
+         circumference * _test[_i][_qp];
```

Comment:
Right, I realized that's it not quite ready for axial application because the view factor type relations are wrong. Really, this is more about switching from perimeters to areas now, since we're working with areas in the component.